CHANGES.txt

```diff
@@ -5,10 +5,20 @@ Changes for crate
 Unreleased
 ==========
 
-- SQLAlchemy: Support ``INSERT...VALUES`` with multiple value sets by enabling
+- SQLAlchemy Core: Support ``INSERT...VALUES`` with multiple value sets by enabling
   ``supports_multivalues_insert`` on the CrateDB dialect, it is used by pandas'
   ``method="multi"`` option
 
+- SQLAlchemy Core: Enable the ``insertmanyvalues`` feature, which lets you control
+  the batch size of ``INSERT`` operations using the ``insertmanyvalues_page_size``
+  engine-, connection-, and statement-options.
+
+- SQLAlchemy ORM: Remove support for the legacy ``session.bulk_save_objects`` API
```

Comment:
Shouldn't we mark this a breaking change? (we can also do during release of course)

Reply:
Thanks. Initially, I had a strong BREAKING CHANGE admonition here. But after observing that the legacy bulk API will still work for users on SA 1.x, and given that support for SA 2.x has only been added recently, I relaxed that note to:
```
Note: The legacy mode will still work on SQLAlchemy 1.x, while SQLAlchemy 2.x users
MUST switch to the new method now.
```
Please let me know if you think we should bring back the stronger note.